### ISSUE TITLE
Back to Normal ? Maybe?

### ISSUE THREAD

**Issue by Ladysarajane** [teagan75/Metamorphoses-2#36]
As life is starting slowly to return to normal here...or what passes for normal, I got some time to play a bit. I started a new world since it had been so long since I played. Guess what!!! Jade is back to normal and gave me the correct turnips. Aubergine seeds are back in the General store too. Seems they were there are along. Maybe Overwolf was not reading the scripts correctly? I am really interested to see if other things are back to normal too. TEST ON!! LOL

**Comment by teagan75** [comment]
I am happy things are getting back to normal :) I am glad to hear stuff is there and working now. It was frustrating me a lot

**Comment by Ladysarajane** [comment]
The weapons rack from the blacksmith are now working correctly. I closed that topic. I am working on cinderbella seeds next to see if that is a problem still.

**Comment by teagan75** [comment]
cinderbella is still a problem and reported to the mod dev

**Comment by Ladysarajane** [comment]
Yep.....couldn't get them to work. I was hoping that overwolf was the cause.
